banners/DE/pi30.json

```
@@ -0,0 +1,25 @@
+{
+    "meta": {
+        "offerCountry": "DE",
+        "offerType": "PI30NQ",
```

```suggestion
        "offerType": "PI30",
```
How about we change this to PI30 for non-qualifying.